src/main/app/src/app/admin/admin/admin.component.ts

```diff
@@ -43,13 +47,19 @@ export abstract class AdminComponent extends ViewComponent {
     this.menu.push(
       this.getMenuLink("title.parameters", "/admin/parameters", "tune"),
     );
-    this.menu.push(
-      this.getMenuLink(
-        "title.formulierdefinities",
-        "/admin/formulierdefinities",
-        "design_services",
-      ),
-    );
+    this.configuratieService
+      .readFeatureFlagBpmnSupport()
+      .subscribe((bpmSupport) => {
```

Comment:
@infonl-marcel I hope this can be improved? With the current implementation what happens is that sometimes this button is rendered above the 'inrichtingscheck' button and sometimes below. This is because of the async subscribe mechanism of course. I just don't know about about this to fix that. ;-)

Reply:
Indeed; it's an async problem. After loading the 'Formulierdefinities' is the latest pushed item in the array. Looks like when clicking the sidebar menu is populated again. But without the endpoint request. Then the order of pushing is different.

Most simple solution since only 1 async menu item is put it last in the menu list. If there will be other async items there needs to be implemented more logic around the menu items to keep the order.

This menu changing effect also appears when changing the Zaak state. Tmho it is a less nice UX. Here the same counts and needs same approach to solve if needed. Will kill many birds with one stone.

Reply:
Ok, clear. Let's leave it for now then. It's a temporary thing anyway.